staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness.go

```diff
@@ -37,6 +37,11 @@ type priorityAndFairnessKeyType int
 
 const priorityAndFairnessKey priorityAndFairnessKeyType = iota
 
+const (
+	responseHeaderMatchedPriorityLevelConfigurationUID = "PriorityLevelConfiguration-UID"
```

Comment:
I don't know if it's local convention to put an X- at the front of nonstandard k8s-specific headers or not?

Reply:
I asked --- https://kubernetes.slack.com/archives/C0EG7JC6T/p1582040901335700 .  No answer yet.

Reply:
Yes, "X-Kubernetes-", in fact. I suggest `X-Kubernetes-PF-PriorityLevelUID` and `X-Kubernetes-PF-FlowSchemaUID`